Workbooks/Individual Storage/Overview/Overview.workbook

```diff
@@ -0,0 +1,402 @@
+    {
+      "type": 10,
+      "content": {
+        "chartId": "workbook85d2d5d7-0e4c-4c24-8d42-47bf8300bdc0",
+        "version": "MetricsItem/1.0",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "resourceType": "microsoft.storage/storageaccounts",
+        "resourceParameter": "StorageAccount",
+        "metrics": [
+          {
+            "namespace": "microsoft.storage/storageaccounts",
+            "metric": "microsoft.storage/storageaccounts-Capacity-UsedCapacity",
+            "aggregation": 4,
+            "chartType": 3,
+            "splitBy": null,
+            "color": "lightBlue"
+          },
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Used capacity",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "*Availability* - the percentage of availability for the storage service. Availability is calculated by taking the total billable requests value and dividing it by the number of applicable requests, including those requests that produced unexpected errors. All unexpected errors result in reduced availability for the storage service or the specified API operation. \r\n<br />\r\n<br />"
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "text - 11"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "*Used capacity* - the amount of storage used by the storage account. For standard storage accounts, it's the sum of capacity used by blob, table, file, and queue. For premium storage accounts and Blob storage accounts, it is the same as BlobCapacity.\r\n<br />\r\n<br />"
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "text - 11 - Copy"
+    },
+    {
+      "type": 10,
+      "content": {
+        "chartId": "workbook85d2d5d7-0e4c-4c24-8d42-47bf8300bdc0",
+        "version": "MetricsItem/1.0",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "resourceType": "microsoft.storage/storageaccounts",
+        "resourceParameter": "StorageAccount",
+        "metrics": [
+          {
+            "namespace": "microsoft.storage/storageaccounts",
+            "metric": "microsoft.storage/storageaccounts-Transaction-SuccessE2ELatency",
+            "aggregation": 4,
+            "chartType": 3,
+            "splitBy": null,
+            "color": "yellow"
+          },
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Success end-to-end latency",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy - Copy"
```

Comment:
add name for this guy "metric - 0 - Copy - Copy"?

Reply:
Fixed